libs/langchain/langchain/vectorstores/redis/schema.py

```diff
@@ -97,9 +97,9 @@ class RedisVectorField(RedisField):
     algorithm: object = Field(...)
     datatype: str = Field(default="FLOAT32")
     distance_metric: RedisDistanceMetric = Field(default="COSINE")
-    initial_cap: int = Field(default=20000)
+    initial_cap: Optional[int] = None
 
-    @validator("distance_metric", pre=True)
+    @validator("algorithm", "datatype", "distance_metric", pre=True, each_item=True)
```

Comment:
What does ``each_item`` do here? I though that was for collections? is that necessary?

Reply:
removing its not needed